### PR TITLE
Updated ExcelActivateCellCommand to ExcelActivateRangeCommand

### PR DESCRIPTION
**Pull request by openbots-aw**
- Changed name and descriptio from cell to range

### CODE REVIEW

File: OpenBots.Commands/OpenBots.Commands.Microsoft/OpenBots.Commands.Excel/ExcelActivateRangeCommand.cs [hunk 1]
```diff
@@ -29,17 +29,17 @@ public class ExcelActivateCellCommand : ScriptCommand
 		public string v_InstanceName { get; set; }
 
 		[Required]
-		[DisplayName("Cell Location")]
-		[Description("Enter the location of the cell to activate.")]
-		[SampleUsage("A1 || {vCellLocation}")]
+		[DisplayName("Cell Range")]
+		[Description("Enter the range of cells to activate.")]
+		[SampleUsage("A1:B10 || A1 || {vCellLocation}")]
```
Comment (openbots-ff) [inline OpenBots.Commands/OpenBots.Commands.Microsoft/OpenBots.Commands.Excel/ExcelActivateRangeCommand.cs:34]: Please refer to ExcelDeleteRange or any of the commands that take a cell/range as an input and copy the display name/sample usage as well as the variable conversion. This input currently only works for variables like {var} and not {var1}:{var2}

**Review by openbots-ff — CHANGES_REQUESTED** [review]

**Review by openbots-ff — CHANGES_REQUESTED** [review]
Please go to OpenBots.Studio > Supplementary Files > Script Conversion Files > 1.2.1.0 and add a conversion to address this file name change: 
"ExcelActivateCellCommand": "ExcelActivateRangeCommand",
"Activate Cell": "Activate Range"

File: OpenBots.Commands/OpenBots.Commands.Microsoft/OpenBots.Commands.Excel/ExcelActivateRangeCommand.cs (hunk 1 above)
Comment (openbots-aw) [reply OpenBots.Commands/OpenBots.Commands.Microsoft/OpenBots.Commands.Excel/ExcelActivateRangeCommand.cs:35]: Done.

Comment (openbots-ff) [reply OpenBots.Commands/OpenBots.Commands.Microsoft/OpenBots.Commands.Excel/ExcelActivateRangeCommand.cs:35]: Sample Usage and Display Name have not been updated as requested

**Review by openbots-ff — CHANGES_REQUESTED** [review]

File: OpenBots.Commands/OpenBots.Commands.Microsoft/OpenBots.Commands.Excel/ExcelActivateRangeCommand.cs [hunk 2]
```diff
@@ -30,9 +30,9 @@ public class ExcelActivateRangeCommand : ScriptCommand
 		public string v_InstanceName { get; set; }
 
 		[Required]
-		[DisplayName("Cell Range")]
-		[Description("Enter the range of cells to activate.")]
-		[SampleUsage("A1:B10 || A1 || {vCellLocation}")]
+		[DisplayName("Range")]
+		[Description("Enter the location of the cell or range to select.")]
```
Comment (openbots-ff) [inline OpenBots.Commands/OpenBots.Commands.Microsoft/OpenBots.Commands.Excel/ExcelActivateRangeCommand.cs:34]: committing change

Comment (openbots-ff) [inline OpenBots.Commands/OpenBots.Commands.Microsoft/OpenBots.Commands.Excel/ExcelActivateRangeCommand.cs:34]:
```suggestion
		[Description("Enter the location of the cell or range to activate.")]
```

**Review by openbots-ff — APPROVED** [review]

**Review by openbots-ch — APPROVED** [review]

**Pull request merged**